npm/translatePolicy.go

```diff
@@ -224,7 +224,9 @@
 			entry.Specs = append(
```

Comment:
Do we need to update the ItablesComment and append - MARK-0x100?

Reply:
I mean we can, but i am worried about the length of a given rule in some instances.